Makefile

```diff
@@ -41,7 +41,7 @@ flake8:
 	flake8 .
 
 unit-test:
-	pytest tools/
+	pytest --disable-warnings tests/
```

Comment:
There a few warnings of deprecation of some libraries, we can get it resolved after closing this